website/_dogfooding/_docs tests/code-wrap-test.md

```diff
@@ -0,0 +1,19 @@
+# Doc with code
```

Comment:
Could you remove this page and add it to here? https://github.com/facebook/docusaurus/blob/main/website/_dogfooding/_pages%20tests/code-block-tests.mdx

Test files are really hard to maintain if they are all over the place

Reply:
Actually we already have a code block on this page that demonstrates the issue and I can see that it's fixed:

<img width="677" alt="image" src="https://user-images.githubusercontent.com/55398995/183296569-ed745a39-6e67-4435-a3f0-c5dac3c0c283.png">

I suggest removing this test. Very much appreciate it that you added it, though.

Reply:
Cool, I will do that.